pkg/resource/model/testdata/test_suite.yaml

```diff
@@ -2,16 +2,173 @@ tests:
   - name: "Model create tests"
     description: "Part of model CRD tests."
     scenarios:
-     - name: "Create=InvalidInput"
-       description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
-       given:
-         desired_state: "model/v1alpha1/m_invalid_before_create.yaml"
-         svc_api:
-           - operation: CreateModelWithContext
-             error:
-               code: InvalidParameterValue
-               message: "The model name must not include a special character."
-       invoke: Create
-       expect:
-         latest_state: "model/v1alpha1/m_invalid_create_attempted.yaml"
-         error: resource is in terminal condition
+      - name: "Create=InvalidInput"
+        description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
+        given:
+          desired_state: "v1alpha1/create/desired/invalid_before_create.yaml"
+          svc_api:
+            - operation: CreateModelWithContext
+              error:
+                code: InvalidParameterValue
+                message: "The model name must not include a special character."
+        invoke: Create
+        expect:
+          latest_state: "v1alpha1/create/observed/invalid_create_attempted.yaml"
+          error: resource is in terminal condition
+      - name: "Create=Valid"
+        description: "Create a new model successfully (ARN in status)."
+        given:
+          desired_state: "v1alpha1/create/desired/success_before_create.yaml"
+          svc_api:
+            - operation: CreateModelWithContext
+              output_fixture: "sdkapi/create/create_success.json"
+        invoke: Create
+        expect:
+          latest_state: "v1alpha1/create/observed/success_after_create.yaml"
```

Comment:
same, filenames should be in pattern with name in line #16

Reply:
couldnt think of better naming but I agree, there should be a pattern to make it more readable